lib/aptible/cli/helpers/operation.rb

```diff
@@ -48,13 +49,53 @@ def cancel_operation(operation)
           operation.update!(cancelled: true)
         end
 
+        def operation_logs(operation)
+          res = get_operation_logs_redirect(operation)
+          s3_file_request = get_operation_logs_s3_file(res.body)
+
+          m = "Printing out results of operation logs for #{operation.id}"
+          CLI.logger.info m
+          puts s3_file_request.body
+        end
+
         def prettify_operation(o)
           bits = [o.status, o.type, "##{o.id}"]
           if o.resource.respond_to?(:handle)
             bits.concat ['on', o.resource.handle]
           end
           bits.join ' '
         end
+
+        private
+
+        def get_operation_logs_redirect(operation)
+          uri = URI(operation.logs_url)
+          headers = { 'Authorization' => "Bearer #{fetch_token}" }
+          http = Net::HTTP.new(uri.host, uri.port)
+          http.use_ssl = true
+          res = http.request(Net::HTTP::Get.new(uri.request_uri, headers))
+          # note: res body with a 200 is target redirect location for download
+          if !res || res.code != '200' || res.body.nil?
+            raise Thor::Error, 'Unable to retrieve the operation\'s logs.'\
```

Comment:
Missing space here and in the test.

```suggestion
            raise Thor::Error, 'Unable to retrieve the operation\'s logs. '\
```